### PR TITLE
Library Management Service initial implementation

### LINKED ISSUE #350: I want to work on Library Management Service

### PR DESCRIPTION
**Pull request by sohan093007**
closes #350

### CODE REVIEW

**Review by sohan093007 — COMMENTED** [review]
Library Management Service initial implementation

File: src/Services/LibraryManagement/LibraryManagement.API/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+                app.UseSwagger();
+                app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "LibraryManagement.API v1"));
```
Comment (NitaiBanik) [inline src/Services/LibraryManagement/LibraryManagement.API/Startup.cs:45]: Can you please fix the indentation here?

File: src/Services/LibraryManagement/LibraryManagement.API/Controllers/HelloWorldController.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+namespace LibraryManagement.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class HelloWorldController : ControllerBase
+    {
+        [HttpGet]
+        public static string Ping()
+        {
```
Comment (NitaiBanik) [inline src/Services/LibraryManagement/LibraryManagement.API/Controllers/HelloWorldController.cs:11]: please return `ActionResult<string>` instaed of `static string`

**Review by sohan093007 — COMMENTED** [review]
Made some changes according to Nitai's Comment.

**Review by sohan093007 — COMMENTED** [review]
LibraryMangment after indentation

**Review by sohan093007 — COMMENTED** [review]
LibraryMangment after indentation

File: src/Services/LibraryManagement/LibraryManagement.API/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            app.UseSwagger();
+            app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "LibraryManagement.API v1"));
+
+            app.UseHttpsRedirection();
```
Comment (ratanparai) [inline src/Services/LibraryManagement/LibraryManagement.API/Startup.cs:47]: remove https redirection.

**Review by ratanparai — CHANGES_REQUESTED** [review]

**Review by sohan093007 — COMMENTED** [review]
After removing HTTPS redirection

**Review by sohan093007 — COMMENTED** [review]
After removing https redirection

**Review by ratanparai — APPROVED** [review]

**Pull request merged**